python/lsst/afw/table/aliasMap.py

```diff
@@ -2,12 +2,20 @@
 
 from ._aliasMap import AliasMap
 
-def _items(self):
-    """
-    pybind11 has a ``make_iterator`` method that greatly reduces the amount of code needed to
-    iterate over an AliasMap and it's keys. This function remains to avoid changes to the AliasMap API
-    but in the future could be removed if desired.
-    """
-    return [a for a in self]
-
-AliasMap.items = _items
\ No newline at end of file
+
+def _keys(self):
+    """Return an iterator over AliasMap keys"""
+    for key, value in self.items():
+        yield key
+
+
+def _values(self):
+    """Return an iterator over AliasMap values"""
+    for key, value in self.items():
+        yield value
+
+AliasMap.__iter__ = _keys
+
+AliasMap.keys = _keys
+
+AliasMap.values = _values
```

Comment:
I'm pretty sure something wasn't working, but I don't remember what it was. Unfortunately I don't remember -- if it was in a commit message that info was lost when I squashed everything.